examples/pytorch/ogb/ngnn_seal/main.py

```diff
@@ -53,11 +48,16 @@ def __init__(
         else:
             self.node_features = None
 
+        if not self.dynamic:
+            self.g_list, tensor_dict = self.load_cached()
+            self.labels = tensor_dict["y"]
+            return
```

Comment:
What are the motivations for changing the location of this logic?

Reply:
Even if we don't use dynamic dataset, we should set self.edge_weights and self.node_features for the subsequent use.